application/controllers/IndexController.php

```diff
@@ -84,15 +84,11 @@ public function newsAction()
         $owFeed  = null;
         $version = $this->_config->version;
 
+        $translate = OntoWiki::getInstance()->translate;
         $this->view->placeholder('main.window.title')->set('News');
 
-        try {
-            $url = 'http://blog.aksw.org/feed/?cat=5&client='
-                . urlencode($version->label)
-                . '&version='
-                . urlencode($version->number)
-                . '&suffix='
-                . urlencode($version->suffix);
+            try {
+            $url = $translate->translate('http://amsl.technology/feed/?lang=en');
```

Comment:
There is a new configurable option for that in OntoWiki 1.0 introduced in https://github.com/AKSW/OntoWiki/pull/341